docs/index.md

```diff
@@ -56,314 +61,6 @@ Because it defaults to accepting `.c` files, we would like to change it to `*.pd
 ### Grading submissions
 Click on `Grade Submissions`, and then the arrow button to open up student submissions. For details on the relevant features for an Instructor, go to [Guide for Instructor](/instructors)
 
-## Getting Started
-
-Autolab consists of two services: (1) the Ruby on Rails frontend, and (2) [Tango](/tango), the RESTful Python autograding server. Either service can run independently without the other. But in order to use all features of Autolab, we highly recommend installing both services.
-
-Currently, we have support for installing Autolab on [Ubuntu 18.04+](#ubuntu-1804), and [Mac OSX](#mac-osx-1011).
-
-The following instructions are for manual installation of Autolab. We strongly encourage you to try out the [Docker Compose](/docker-compose) installation method which is much faster and easier. It is production ready and comes deployed with MySQL and TLS.
-
```

Comment:
I recommend saying MySQL database and SSL/TLS (just in case people aren't sure what TLS is)